### DISCUSSION TITLE
How do I browse the Icons?

### DISCUSSION THREAD

**Discussion by ultrazebs** [MahApps/MahApps.Metro.IconPacks#312] [General]
I always see the green window with all the icons, but how do I get that window?

I've been going crazy over this. What do I have to launch? WHat do I have to download?

**Comment by punker76** [comment]
@ultrazebs You can use the [IconPacks.Browser](https://github.com/MahApps/IconPacks.Browser) which is also mentioned in the [README](https://github.com/MahApps/MahApps.Metro.IconPacks?tab=readme-ov-file#iconpacks-browser)
